gen/src/reader/ucd/unicode_data.rs

```diff
@@ -175,15 +182,10 @@ pub struct UnicodeDataEntry {
     // by unsafe-hacking a u32::MAX char, and not losing any application space.
 }
 
+
 #[derive(Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
-pub struct UnicodeData(Box<[UnicodeDataEntry]>);
+pub struct UnicodeData(pub Box<[UnicodeDataEntry]>);
 
-impl UnicodeData {
-    pub fn iter(&self) -> Iter<UnicodeDataEntry> {
```

Comment:
Why did you remove this fn? It's much clearer at the use site to call `UNICODE_DATA.iter()` rather than `UNICODE_DATA.0.iter()`.

I say use the following:

```rust
pub struct UnicodeData(Box<[UnicodeDataEntry]>);
impl UnicodeData {
    pub fn iter(&self) -> Iter<UnicodeDataEntry> {
        self.0.iter()
    }
}
```

It's just much cleaner not to have to `.0.iter()` at the use site.

Reply:
Of course, then you can bring up the other ones that wrap a `To*DataTable`. Honestly, I don't know. You could to a pass-through impl of that trait there as well, but that unnecessary code just shouldn't be required in this script environment. So I don't really have an opinion on this one, I've decided.

Reply:
For all the data models under `reader`, I've made the internals public, so we don't have to write and maintain wrappers. I'll explain more in the follow up.